src/features/utils/instrument-base.js

```diff
@@ -42,16 +44,26 @@ export class InstrumentBase extends FeatureBase {
   importAggregator () {
     if (this.hasAggregator || !this.auto) return
     this.hasAggregator = true
-
+    let session, agentSessionFile
+    if (getConfigurationValue(this.agentIdentifier, 'privacy.cookies_enabled') === true) {
```

Comment:
We really should create a new setting for this instead of overloading the meaning of `cookies_enabled`.

Reply:
yeah this setting is obscured rn.  In the UI, it is represented a "session tracking" but passes through the config as "cookies_enabled"